rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -25,6 +25,10 @@ while /bin/true; do
     && pkill -9 --ns 1
   sleep 20
 done &
+sleep 30
```

Comment:
Why sleeping 30 seconds can solve the problem? How shall we choose the length of the time?

Reply:
Hi Yifan, have you tried to comment out the backend while cluse and sleep to see whether we can get classpath without sleeping while pulling and running jobs together?

---
In reply to: [161923022](https://github.com/Microsoft/pai/pull/235#discussion_r161923022) [](ancestors = 161923022)